controllers/API/school.js

```diff
@@ -17,39 +17,28 @@ router.post("/API/school", auth.isAuthorised("ALTER_SCHOOLS"), function (req, re
 		if (err) {
 			logger.warning("Can not add school\n" + err);
 			res.redirect("/options");
-			res.sendStatus(400);
 		} else {
 			res.redirect("/options");
-			res.sendStatus(200);
 		}
 	});
 
 });
 
 router.get("/API/school", function (req, res) {
-	if (req.query.id) {
-		School.findById(req.query.id, function (err, school) {
-			if (typeof school === "undefined" || err) {
-				logger.warning("Can not find school\n" + (err || "Wrong id provided"));
+	School
+		.find(req.query)
+		.limit(parseInt(req.query.count) || 10)
+		.exec(function (err, schools) {
+			if (err) {
+				logger.warning("Can not retrieve schools\n" + err);
 				res.sendStatus(400);
 			} else res.send(schools);
 		});
-	} else {
-		School
-			.find({})
-			.limit(req.query.count || 10)
-			.exec(function (err, schools) {
-				if (err) {
-					logger.warning("Can not retrieve schools\n" + err);
-					res.sendStatus(400);
-				} else res.send(schools);
-			});
-	}
 });
 
 // delete a school
 router.delete("/API/school", auth.isAuthorised("ALTER_SCHOOLS"), function (req, res) {
-	School.findOneAndRemove({"_id": req.body.id}, function (err) {
+	School.findOneAndRemove({ "_id": req.body.id }, function (err) {
```

Comment:
Is this whitespace something your IDE automatically inserts? 
If you actually want that, then the following line should be added to  `.eslintrc.js`:

`"object-curly-spacing": ["error", "always"]`

Reply:
no that was a mistake from me.

Reply:
Either way, our ESlint isn't enforcing any convention regarding that style right now. We should probably add a rule that forces either a space after and before a curly bracket, or force that no spaces should be there.